api/routes/docs/get.js

```diff
@@ -0,0 +1,20 @@
+const logger = require('../../logger')('document routes');
+const { getFile: get } = require('../../files');
+
+const HELP_DOC = 'EUAAccountRegistration.pdf';
+
+module.exports = (app, { getFile = get } = {}) => {
+  logger.silly('setting up GET /docs/account-registration route');
+
+  app.get('/docs/account-registration', async (req, res) => {
+    try {
+      const file = await getFile(HELP_DOC);
+      res.setHeader('Content-Type', 'application/octet-stream');
+      res.setHeader('Content-Disposition', `attachment; filename=${HELP_DOC}`);
+      res.send(file).end();
+    } catch (e) {
+      logger.error({ id: req.id, message: 'error fetching file' });
+      res.status(400).end();
+    }
```

Comment:
Instead of a `try...catch` block, you could do it the "promise" way. See `api/routes/states/get.js` for an example.

Not a necessary change, but it would be nice to unify the api code to a particular style.

Reply:
I updated it, if you want to take a look again